backend/src/database/repositories/incomingWebhookRepository.ts

```diff
@@ -232,4 +232,27 @@ export default class IncomingWebhookRepository extends RepositoryBase<
       type: QueryTypes.DELETE,
     })
   }
+
+  async checkWebhooksExistForIntegration(integrationId: string): Promise<boolean> {
+    const transaction = this.transaction
+
+    const results = await this.seq.query(
+      `
+      select count(*) as count
+      from "incomingWebhooks"
+      where "integrationId" = :integrationId
+      limit 1
+    `,
+      {
+        replacements: {
+          integrationId,
+        },
+        type: QueryTypes.SELECT,
+        transaction,
+      },
+    )
+
+    // @ts-ignore
+    return results[0].count > 0
```

Comment:
watch out here - `count` returns a `bigint` in postgres and some javascript ORMs return `string` instead of a `number` for this case. Better double check to see if it's a number or a string that you have to `parseInt`.

Reply:
I changed to this

```
async checkWebhooksExistForIntegration(integrationId: string): Promise<boolean> {
    interface QueryResult {
      count: number
    }

    const transaction = this.transaction

    const results: QueryResult[] = await this.seq.query(
      `
      select count(*)::int as count
      from "incomingWebhooks"
      where "integrationId" = :integrationId
      limit 1
    `,
      {
        replacements: {
          integrationId,
        },
        type: QueryTypes.SELECT,
        transaction,
      },
    )

    return results.length > 0 && results[0].count > 0
  }```